load-jetpack.php

```diff
@@ -74,8 +74,6 @@ function jetpack_should_use_minified_assets() {
 
 add_action( 'updating_jetpack_version', array( 'Jetpack', 'do_version_bump' ), 10, 2 );
 add_action( 'init', array( 'Jetpack', 'init' ) );
-add_action( 'plugins_loaded', array( 'Jetpack', 'plugin_textdomain' ), 99 );
```

Comment:
This is changing the priority of textdomian loading to 90 (via late_initialization). Does it matter?

Reply:
Fixed that in the next commit, thanks!